playbook.md

```diff
@@ -971,189 +971,72 @@ to build an appropriate hosting platform for their service. This is fair
 enough, because they shouldn't have to. We built GovPress to meet this need,
 and it is now used by most of our clients.
 
+We build [websites](https://www.digitalmarketplace.service.gov.uk/g-cloud/services/930612236449495), [campaign](https://www.digitalmarketplace.service.gov.uk/g-cloud/services/691308552308120) and [blogging](https://www.digitalmarketplace.service.gov.uk/g-cloud/services/355674790119695) platforms hosted on GovPress, configured to our client’s branding requirements.
```

Comment:
I'd rephrase this a little - the service offer is for managed hosting and support rather than website build. Maybe something like 'We offer managed hosting and support for WordPress websites, as well as blogging and campaigns platforms, configured to our client's branding requirements, all on our GovPress hosting platform. 

Wonder if it's worth saying that we offer these services through the GCloud framework (as that's where the links are to)?